stock_request/models/stock_request.py

```diff
@@ -5,13 +5,26 @@
 from odoo.exceptions import UserError, ValidationError
 from odoo.tools import float_compare
 
+REQUEST_STATES = [
+    ("draft", "Draft"),
```

Comment:
If you don't use selection attribute directly, you should use the translate function.

But I see this approach has been finally removed : https://github.com/OCA/stock-logistics-warehouse/pull/1622/files

So, please get the previous code back

Reply:
The same stated above.

Reply:
@nicomacr Please changes this according to what pointed above:

![image](https://user-images.githubusercontent.com/19529533/232504096-82c19201-a6d5-405d-871b-e056aade735f.png)